content/en/docs/Kyverno CLI/_index.md

```diff
@@ -224,3 +224,168 @@ policies:
 ```
 kyverno apply /path/to/add_network_policy.yaml --resource /path/to/required_default_network_policy.yaml -f /path/to/value.yaml
 ```
+
+#### Policy Report
+
+Policy reports provide information about policy execution and violations. Use '--policy_report' with the apply command to generate policy report.
+
+Policy report can also be generated for a live cluster. While generating policy report for a live cluster the '-r' flag is assuming a resource by specific name which is assumed to be globally unique. And it doesn't support naming the resource type (ex., Pod/foo when the cluster contains resources of different types with the same name). To generate policy report for a live cluster use '--cluster' with '--policy_report'.
```

Comment:
"Policy reports" (reports plural)
"While generating a policy report for a live cluster..." (nouns as objects almost always have articles, in this case it's "a")
"To generate a policy report..."
`--cluster`
`--policy_report`